experimental/flow/step.go

```diff
@@ -80,9 +81,13 @@ func (s Step) Next(name Name) Step {
 	return s
 }
 
-func (s Step) WithImage(pluginURL, path string) Step {
-	if path != "" {
-		s.template.ImageURL = pluginURL + "/" + strings.TrimPrefix(path, "/")
+func (s Step) WithImage(imageURL string) Step {
+	if u, err := url.Parse(imageURL); err == nil {
```

Comment:
Nit: Should an error be logged if `err != nil`? Return one would make the API hard to use.

Reply:
Logging here is a bit hard, `Step` is a simple struct, doesn't have a logger. 1/5 just ignore invalid URLs... they'd look broken in the UI anyway, easy to spot.

Reply:
:+1: